### ISSUE TITLE
TypeError: 'NoneType' object has no attribute '__getitem__'

### ISSUE THREAD

**Issue by mrmartineau** [sballin/alfred-search-notes-app#3]
I just downloaded the package from Packal but it does not work, here's the debug data:

```sh
[2018-08-10 00:07:22][ERROR: input.scriptfilter] Code 1: Traceback (most recent call last):
  File "/Users/stevejobs/path/to/Alfred.alfredpreferences/workflows/user.workflow.43CB9AFF-A420-4B19-8C99-C393EC988448/searchNotes.py", line 35, in <module>
    for m in matches]
TypeError: 'NoneType' object has no attribute '__getitem__'
```

Let me know if you need me to provide you with more info. FYI, I am on High Sierra at the moment too

**Comment by mrmartineau** [comment]
FYI, I use the code in @richvreeland's [PR](#2) and the workflow now works.

**Comment by sballin** [comment]
Fixed as of v1.1. Added an extra check to make sure the note body is not None, which could also lead to this error.
